### PR TITLE
fix(toolbar): Load toolbar only in body for turbolink

### PR DESCRIPTION
**Pull request by neilkakkar**
## Changes

Restricts toolbar loading only to the body

Removes out of date (thanks to pnpm) local dev instructions.

Fixes https://github.com/PostHog/posthog/issues/13477

---

Ran locally, see the toolbar load in body, works without issues.

...

## Checklist
- [ ] Tests for new code (see [advice on the tests we use](https://github.com/PostHog/posthog-js#tiers-of-testing))
- [ ] Accounted for the impact of any changes across different browsers

### CODE REVIEW

File: src/autocapture-utils.ts [hunk 1]
```diff
@@ -303,14 +303,18 @@ export function isAngularStyleAttr(attributeName: string): boolean {
...
+    scriptUrlToLoad: string,
+    callback: (event: Event) => void,
+    restrictToHTMLBody?: boolean
+): void {
     const scriptTag = document.createElement('script')
     scriptTag.type = 'text/javascript'
     scriptTag.src = scriptUrlToLoad
     scriptTag.onload = callback
 
     const scripts = document.getElementsByTagName('script')
```
Comment (mariusandra) [inline src/autocapture-utils.ts:316]:
```suggestion
    const scripts = document.querySelectorAll('body > script')
```

Chang this line, and I think everything else will work without any other changes. I don't think we need to provide a new parameter, as by default script tags do get added into body.

**Review by mariusandra — COMMENTED** [review]
There's a suggestion inline. Can we do without the new parameter, and just always append to the body? 🤔

**Comment by neilkakkar** [comment]
Works for me, this would be my preferred solution too! I just wasn't sure if there's a specific reason why some might go into the head, and didn't get a response when I asked about it :P - specifically with the recording script 🤷

**Comment by neilkakkar** [comment]
Maybe @benjackwhite can confirm there's no issues?

**Comment by mariusandra** [comment]
Yeah, for the toolbar, it really doesn't matter. The script that loads adds a `<div>` to the end of `<body>`, which then does everything else in a shadow root...  So if we can simplify the code here and the toolbar keeps working, great!

**Comment by neilkakkar** [comment]
Ah, to clarify, the same code path is used for the session recording script, and _I imagine_ it being in the head means it can be loaded slightly faster which might make a difference, idk 🤷 , which is why I didn't want to prematurely change recording behaviour too.

**Comment by benjackwhite** [comment]
Bit late but - I haven't touched a lot of the code related to the recorder loading. From my point of view though I don't see anything concerning here 👍

**Comment by neilkakkar** [comment]
Sounds good, will update it to what Marius suggests then!

File: README.md [hunk 2]
```diff
@@ -29,24 +29,18 @@ Testing on IE11 requires a bit more setup.
...
 1. Run `PostHog/posthog` locally
-2. Link the `posthog-js` dependency to your local version (see above)
+2. Link the `posthog-js` dependency to your local version (see below)
 3. Run `yarn serve` in `posthog-js`. (This ensures `dist/array.js` is being generated)
 4. In your locally running `PostHog/posthog` build, run `yarn copy-scripts`. (This copies the scripts generated in step 3 to the static assets folder for `PostHog/posthog`)
 
 Further, it's a good idea to modify `start-http` script to add development mode: `webpack serve --mode development`, which doesn't minify the resulting js (which you can then read in your browser).
 
-### Using Yalc (Alternative to yarn link)
+### Using Yalc to link local packages
```
Comment (Twixes) [inline README.md:43]: Does yalc work with pnpm?

Comment (mariusandra) [reply README.md:43]: it works

**Review by mariusandra — APPROVED** [review]

**Pull request merged**